soroban-env-common/env.json

```
@@ -2403,6 +2403,32 @@
                     ],
                     "return": "Void",
                     "docs": "Authorizes sub-contract calls for the next contract call on behalf of the current contract. Every entry in the argument vector corresponds to `InvokerContractAuthEntry` contract type that authorizes a tree of `require_auth` calls on behalf of the current contract. The entries must not contain any authorizations for the direct contract call, i.e. if current contract needs to call contract function F1 that calls function F2 both of which require auth, only F2 should be present in `auth_entries`."
+                },
+                {
+                    "export": "4",
+                    "name": "get_address_from_muxed_address",
+                    "args": [
+                        {
+                            "name": "muxed_address",
+                            "type": "MuxedAddressObject"
+                        }
+                    ],
+                    "return": "AddressObject",
+                    "docs": "Returns the address corresponding to the provided MuxedAddressObject as a new AddressObject. Note, that MuxedAddressObject consists of the address and multiplexing id, so this conversion just strips the multiplexing id from the input muxed address.",
+                    "min_supported_protocol": 23
+                },
+                {
+                    "export": "5",
+                    "name": "get_mux_id_from_muxed_address",
```

I don't think we often use the term "mux". It's superfluous here, we just refer to it as the "id" in the XDR. Alternatively we could use the term "muxed_id", but again seems superfluous.
```suggestion
                    "name": "get_id_from_muxed_address",
```

---

Makes sense, just `id` seems good enough.